src/main/java/com/unstoppabledomains/util/Utilities.java

```diff
@@ -8,8 +8,11 @@
   private Utilities() {}
 
   public static boolean isNull(String value) {
-    return ( value == null || value.equals("0x0000000000000000000000000000000000000000")
-        || value.equals("") || value.length() == 0);
+    return ( value == null ||
+    value.equals("0x0000000000000000000000000000000000000000") ||
+    value.equals("0x0") ||
```

Comment:
Is this a code for a general error? If so, I think we should handle this separately in a place where we have more context about the root cause.

Reply:
E.g.: `Contract#isUnknownError`.

Reply:
This is more of a utillity function to check if the response is empty. I renamed the function in #16 to reduce confussion